client/src/common/Sidebar.jsx

```diff
@@ -0,0 +1,47 @@
+import { observer } from 'mobx-react-lite';
+import React, { useContext } from 'react';
+import HamburgerMenu from 'react-hamburger-menu';
+import { RootStoreContext } from '../stores/stores';
+
+export const styles = {
+  bmMenuWrap: {
+    position: 'fixed',
+    height: '100%',
+  },
+  bmMenu: {
```

Comment:
where are we using these

Reply:
They are CSS classes specific for the hamburger menu react library.